plugins/europeana/search.js

```diff
@@ -52,7 +54,7 @@ function display(field) {
 function fieldsForSearchResult(item) {
   let fields = {
     // TODO: fallback to description when API returns dcDescriptionLangAware
-    dcTitle: item.dcTitleLangAware ? display(item.dcTitleLangAware) : `No title provided for record ID ${item.id}`,
+    dcTitle: item.dcTitleLangAware ? display(item.dcTitleLangAware) : `${$t('messages.noTitle')}: ${item.id}`,
```

Comment:
Use [named formatting](http://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) for the item ID.